app.py

```diff
@@ -171,4 +174,6 @@ def _start_screeningroom_rv(self):
                 base_url=self.shotgun.base_url, context=entity, path_to_rv=rv_path
             )
         except Exception as e:
-            self.log_error("Could not launch RV Screening Room. Error reported: %s" % e)
+            self.logger.error(
```

Comment:
Mind adding in a `self.logger.exception(e)` here? As it stands, we won't get a traceback. You can keep the `self.logger.error` call. There's no harm in having both.

Reply:
good idea!